image/presto-entrypoint.py

```diff
@@ -116,6 +117,16 @@ def bootstrap(arguments: dict):
             custom_directory=TEMPLATE_CUSTOM_DIRECTORY,
             **arguments
         )
+    ## Render custom configs that don't have a default template
```

Comment:
[PEP 8 block comment](https://www.python.org/dev/peps/pep-0008/#id31) format:

```suggestion
    # Render custom configs that don't have a default template
```